### DISCUSSION TITLE
Source error 问题，可能是因为科学上网插件错误的DNS分流配置所造成

### DISCUSSION THREAD

**Discussion by zycboss** [xiaye13579/BBLL#529] [Show and tell]
客户端是大法的X9000F电视，家里路由器上配置了passwall科学上网，使用xray分流

passwall的DNS设置选项`当使用中国列表外时的默认DNS`设置为`远程DNS`时，会导致90%以上的视频播放时出现黑屏 `source error` 和`null`
将此选项更改为`直连DNS`后，所有视频均可正常加载播放

所以，可能与解码器无关，有人说的看一会直播，并不能解决因为DNS分流错误导致的source error 的问题

各位遇到类似问题的可以检查一下自己科学上网插件对应的设置

**Comment by GDZJF** [comment]
那该怎么设置？我家电视很垃圾

**Comment by GDZJF** [comment]
我家的电视是夏普，该怎么办？

**Reply by adidalin** [reply]
没有用上网插件，同样不能用

**Reply by GDZJF** [reply]
那该怎么解决呢？

**Reply by GDZJF** [reply]
怎么解决？

**Comment by qiji2024** [comment]
会不会是清晰度问题，这个没有软件没有480p，所以会显示 source error
